code/modules/reagents/chemistry/reagents/drugs.dm

```diff
@@ -723,18 +817,28 @@
 	overdose_threshold = 20
 	addiction_chance = 10
 	addiction_threshold = 5
+	addiction_decay_rate = 0.2
 	taste_description = "silicon"
 
 
 /datum/reagent/surge/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
+	var/recent_consumption = holder.addiction_threshold_accumulated[type]
 	M.Druggy(30 SECONDS)
+	M.AdjustParalysis(-4 SECONDS)
+	M.AdjustStunned(-4 SECONDS)
+	M.AdjustWeakened(-4 SECONDS)
+	update_flags |= M.adjustStaminaLoss(-25, FALSE)
 	var/high_message = pick("You feel calm.", "You feel collected.", "You feel like you need to relax.")
-	if(prob(1))
-		if(prob(1))
-			high_message = "01010100010100100100000101001110010100110100001101000101010011100100010001000101010011100100001101000101."
+	if(prob(10))
+		high_message = "0100011101001111010101000101010001000001010001110100111101000110010000010101001101010100!"
 	if(prob(5))
 		to_chat(M, "<span class='notice'>[high_message]</span>")
```

Comment:
move `var/high_message` definition inside this if; it's not used outside of it.
(only mentioning this because you dared bugfix the `if(prob(1)) if(prob(1))` chain above; Feel free to leave as is)